net/netifyd/files/netifyd.init

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh /etc/rc.common
+# Netify Agent
+# Copyright (C) 2016-2018 eGloo, Incorporated
+#
+# This is free software, licensed under the GNU General Public License v3.
+
+START=50
+STOP=50
+
+USE_PROCD=1
+PROG=/usr/sbin/netifyd
+NETIFYD_AUTODETECT=yes
+
+start_service() {
+    mkdir -p /var/lib/netifyd
+    mkdir -p /var/run/netifyd
+    if [ -f /usr/libexec/netifyd/functions.sh ]; then
+        source /usr/libexec/netifyd/functions.sh
+        NETIFYD_OPTS=$(auto_detect_options)
+
+        load_modules
+    fi
+
+    procd_open_instance
+    procd_set_param command $PROG -R $NETIFYD_OPTS
+    procd_set_param file /etc/netifyd.conf
+    procd_set_param respawn
+    procd_close_instance
+}
+
+shutdown() {
+    stop
```

Comment:
why do you need this hook explicitly? you should be able to just drop this right?